core/domain/stats_jobs_one_off.py

```diff
@@ -303,6 +315,25 @@ def reduce(exp_id, values):
 
     @classmethod
     def prepare_reduce(cls, exp_id, values):
+        """Runs the reduce step to extract aggregate exploration statistics from
+        the dicts passed in by the mapping function.
+
+        Args:
+            exp_id: str. The exploration id.
+            values: list(dict(str, str)). List of dicts of events for preparing
+                the reduce.
+
+        Returns:
+            tuple(list(dict), list(ExplorationStats), list(str)). 3-tuple where:
+            - The first element is the list of dicts of different version-wise
+                ExplorationStats. Each dict contains the following key-value
+                pairs:
+            - The second element is the list of ExplorationStats domain class
+                instances of the corrupted statistics models corresponding to
+                the specified version.
```

Comment:
When you say "the specified version", what are you referring to?

Reply:
I was referring to version 1 or 2 of the model.